staging/src/k8s.io/api/auditregistration/v1alpha1/types.go

```diff
@@ -90,41 +90,68 @@ type AuditSinkList struct {
 	// +optional
 	metav1.ListMeta `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
 
-	// List of audit configurations.
+	// `items` is a list of `auditSink` configurations.
 	Items []AuditSink `json:"items" protobuf:"bytes,2,rep,name=items"`
 }
 
 // Policy defines the configuration of how audit events are logged
 type Policy struct {
-	// The Level that all requests are recorded at.
+	// `level` is the base level that all requests are recorded at.
 	// available options: None, Metadata, Request, RequestResponse
 	// required
 	Level Level `json:"level" protobuf:"bytes,1,opt,name=level"`
 
-	// Stages is a list of stages for which events are created.
+	// `stages` is a list of stages for which events are created.
+	// If no stages are given nothing will be logged
```

Comment:
If no stages have been specified, nothing will be logged even if the rules specify stages of their own?
Can we make any correlation more explicit if there is one?